pkg/skaffold/errors/errors_test.go

```diff
@@ -243,6 +243,22 @@ func TestShowAIError(t *testing.T) {
 				Suggestions: reportIssueSuggestion(config.SkaffoldOptions{}),
 			},
 		},
+		{
+			description: "deploy failed",
+			opts:        config.SkaffoldOptions{},
+			context:     &config.ContextConfig{},
+			phase:       Deploy,
+			err:         fmt.Errorf(`exiting dev mode because first deploy failed: unable to connect to Kubernetes: Get "https://192.168.64.3:8443/version?timeout=32s": net/http: TLS handshake timeout`),
```

Comment:
We should present the cluster ip and also the error details to the user.
e.g.

```
Deploy Failed. Could not connect to cluster gke_tejal-test_us-central1-a_dump  at https://192.168.64.3:8443 due to TLS handshake timeout. Check your cluster connection
```

Reply:
Fixed.